gallery/src/styles/NxList/NxListErrorStateExample.tsx

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, {useState} from 'react';
+import { NxLoadError } from '@sonatype/react-shared-components';
+
+function NxListErrorExample() {
+  const [error] = useState<string | null>('Server Error');
+
+  return (
+    <ul className="nx-list">
+      <li className="nx-list__item nx-list__item--error">
+        <NxLoadError error={error} retryHandler={() => {}} />
```

Comment:
You don't really need the `useState` and stuff in this example, you could just have `error="Server Error"` right here.

Reply:
> You don't really need the `useState` and stuff in this example, you could just have `error="Server Error"` right here.

Hmm, ok, I'm sure I tried that and got an error before probably missed something simple bc it works here: https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/c7f9e80fc6be1e720afdb4dab7e7aa081ef1a42d